notebooks/recommender-system.ipynb

```diff
@@ -0,0 +1,843 @@
+{
```

Comment:
<p>link for <code>*.npy</code>  -&gt; https://numpy.org/doc/stable/reference/generated/numpy.save.html</p><p> </p><blockquote>The dataset used in this tutorial is a subset of the <a href="https://grouplens.org/datasets/movielens/100k/" target="_blank">MovieLens dataset</a>. It consists of 90,570 ratings (1-5) from 943 users on 1680 movies. </blockquote><p>Good place to describe the data row = movie, column = user, rating = 0 (no review), 1-5 is reviewed. </p><p> </p><p>maybe a minimal example:</p><p>|movie|Joe|Jane|Bob|</p><p>|---|---|---|---|</p><p>Jaws|0 | 4 |1|</p><p>Godfather| 5 | 0 | 4</p><p>Star Wars |0 | 5 | 2</p><p> </p><blockquote>This is precisely why we will create a model-based Collaborative Filtering system.</blockquote><p>first mention of model-based <em>Collaborative</em> filtering. Helps to keep vocab consistent throughout i.e. model-based recommendation system. Is the "Collaborative filtering" part of the original </p><p> </p><p><s>First, let us begin</s> Begin by ...</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/cooperrc/uconn_numpy_tutorials/pull/11/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>